code/modules/mob/living/silicon/silicon.dm

```diff
@@ -321,6 +322,15 @@
 
 	usr << browse(list, "window=laws")
 
+/mob/living/silicon/proc/ai_roster()
+	var/dat = "<html><head><title>Crew Roster</title></head><body><b>Crew Roster:</b><br><br>"
+
+	dat += GLOB.data_core.get_manifest()
+	dat += "</body></html>"
+
+	src << browse(dat, "window=airoster")
+	onclose(src, "airoster")

```

Comment:
Why are you setting onclose here if it won't do anything, should be browser not pure browse anyway.

Reply:
This is something I moved from silicon/ai.dm so the pAI could also use it.